components/Layout/layout.module.scss

```diff
@@ -2,8 +2,7 @@
 
 .layout {
   background-color: $techSwitchBlack;
-  width: 100%;
-  height: 100vh;
```

Comment:
did we mean to get rid of this?
I thought this was what ensured that we filled the page?

Reply:
Seems not to affect it anymore but not sure why? Should I put these back in?

Reply:
On this page there could be enough height to take up 100% anyway.

If you zoom out, does it start to break or stay fine?
If it stays fine and we don't need it (on this page and other pages) then that's fine :)

Reply:
no problems when I zoom out  :)